doc/vm-creation-example.md

```diff
@@ -0,0 +1,103 @@
+# Example - Allocating MAC Addresses to a VirtualMachine
+
+## Create a network-attachment-definition:
+
+The 'NetworkAttachmentDefinition' is used to setup the network attachment, i.e. secondary interface for the VirtualMachine.
+For more information see [Kubernetes Network Custom Resource Definition De-facto Standard](https://docs.google.com/document/d/1Ny03h6IDVy_e_vmElOqR7UdTPAG_RNydhVE1Kx54kFQ/edit).
+This example will use a linux-bridge.
+
+```yaml
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+ name: br1
+spec:
+  config: '{
+    "cniVersion": "0.3.1",
+    "name": "br1",
+      "plugins": [
+        {
+          "type": "bridge",
+          "bridge": "br1"
+        },
+        {
+            "type": "tuning"
+        }
+      ]
+    }'
+```
+
+**Note** the tuning plugin changes the MAC Address after the main plugin was executed. Make sure
+the main plugin configure does not have MAC filter on the interface.
+
+## Allocate a MAC Addresses to a VirtualMachine
+
+Create the VirtualMachine with 1 secondary NIC:
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: samplevm
+spec:
+  running: false
+  template:
+    spec:
+      domain:
+        devices:
+          disks:
+          - disk:
+              bus: virtio
+            name: rootfs
+          - disk:
+              bus: virtio
+            name: cloudinit
+          interfaces:
+          - name: default
+            masquerade: {}
+          - bridge: {}
+            name: br1
+        resources:
+          requests:
+            memory: 64M
+      networks:
+      - name: default
+        pod: {}
+      - multus:
+          networkName: br1
+        name: br1
+      volumes:
+        - name: rootfs
+          containerDisk:
+            image: kubevirt/cirros-registry-disk-demo
+        - name: cloudinit
+          cloudInitNoCloud:
+            userDataBase64: SGkuXG4=
+```
+
+Check VirtualMachine:
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  annotations:
+    kubevirt.io/latest-observed-api-version: v1
+    kubevirt.io/storage-observed-api-version: v1
+  name: samplevm
+spec:
+  domain:
+...
+    devices:
+...
+      interfaces:
+      - macAddress: "02:00:00:00:00:09"
+        masquerade: {}
+        name: default
+      - bridge: {}
+        macAddress: "02:00:00:00:00:0a"
+...
+```
+
+**Note** Due to current [issue](https://github.com/k8snetworkplumbingwg/kubemacpool/issues/140), you may encounter 
+issues when starting the vm. A suggested workaround is to disable the pod handler in your namespace. For more information
```

Comment:
pod handler does not ring a bell - maybe MAC address allocation for pods?

Reply:
Done